### DISCUSSION TITLE
deprecation of `quoteIdentifiers` with Snowflake dialect

### DISCUSSION THREAD

**Discussion by nllahat** [sequelize/sequelize#16457] [Help & Questions]
Hi @WikiRik ,

regarding your comment here https://github.com/sequelize/sequelize/pull/2406#issuecomment-1706116800
I wonder how the deprecation of `quoteIdentifiers` will work together with Snowflake since the naming there is all uppercase.
Without the `quoteIdentifiers: false` option - I need to rename all model fields to the uppercase form or use the `field` prop to map to the uppercased column.

Is that right ? am I missing something ?

thank you!!

**Comment by WikiRik** [comment]
My knowledge of snowflake is minimal and we do not the possibility to run integration tests for it so all our support comes from the community. It seems we likely need to update the code mentioned below and document the migration path once the quoteIdentifiers option has been removed. If that will be a renamed snowflake only option or making use of the `field` prop I cannot say at this moment.

https://github.com/sequelize/sequelize/blob/62f0419b2de87a0fbfccc63de4a807681df92f01/packages/core/src/dialects/snowflake/query.js#L117-L143

Do note that v7 already has some major internal changes that have not been tested properly on snowflake so there's much more work to be done. See https://github.com/sequelize/sequelize/discussions/15099 as well
